### ISSUE TITLE
Table Field : Max / Min not allowing empty submission.

### ISSUE THREAD

**Issue by frankkepler** [verbb/formie#945]
**Description**

On the Table Field, Max and Min are acting a bit strange. If you add a Max value, it will not allow an empty field to be passed through, it responds with the error: _'must have one item'_.

The same occurs if you add a value of 0 to the min value.

The visual of the field in the form builder also gets a bit weird (not a huge deal).

**Steps to reproduce**
1. Create a form with a table field, add a max value (with/without min value)
2. Attempt to submit form/page with the table field on it that is empty.

### Form settings
Please pick from the options below.

- Multi-page form: Yes
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: Yes

**Additional info**
- Plugin version: 1.6.1
- Craft version: 3.7.44
- Multi-site: No

**Additional context**
Thanks for all the hard work.

**Comment by engram-design** [comment]
Should be fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.6.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.6.2](https://github.com/verbb/formie/releases/tag/1.6.2)
